packages/x-components/src/design-system/components/suggestion/default.tokens.scss

```diff
@@ -15,6 +15,19 @@
   --x-size-padding-left-suggestion-default: 0;
   --x-size-gap-suggestion-default: var(--x-size-base-03);
 
+  // border
```

Comment:
Same as the suggestion-group. You have to use these new tokens.
And add the border color too.